site/blog/2022/v2.10.1.md

```diff
@@ -0,0 +1,34 @@
+---
+title: v2.10.1
+date: 2022-09-14
+tags:
+    - releases
+---
+
+## VisiData v2.10.1 Released
+
+Hi, data friends!
+
+It’s time for a [new patch release](https://github.com/saulpw/visidata/releases/tag/v2.10.1) with some important fixes and a few minor improvements.
+
+### Fixes
+
+- Macros weren’t working; now they are!
+- Repeat commands were broken, and now are fixed.
+- The Guix build is now working again.
```

Comment:
I would phrase this as "changes necessary for the Guix build have been merged"? Or something like that! It is not that the Guix build was "broken", but that the Guix build maintainer could not build VisiData.